src/map/utils/battleutils.cpp

```diff
@@ -3089,6 +3100,16 @@ namespace battleutils
             * (100 + PAttacker->getMod(Mod::SKILLCHAINBONUS)) / 100
             * (100 + PAttacker->getMod(Mod::SKILLCHAINDMG)) / 100);
 
+        auto PChar = dynamic_cast<CCharEntity *>(PAttacker);
+        if (PChar and PChar->GetMJob() == JOB_NIN and PChar->GetMLevel() >= 75)
+        {
+            auto angle = PDefender->loc.p.rotation - getangle(PDefender->loc.p, PChar->loc.p);
+            // assuming default tolerance of 42 from lua_baseentity.cpp
+            if (angle > 86 and angle < 170)
```

Comment:
this isn't lua. That's not a boolean and. You want &&

This leads me to believe you haven't tested this PR because this would throw a compiler error.

Reply:
I have tested this, that will compile...

Reply:
![image](https://user-images.githubusercontent.com/11480851/72111297-92d58880-32f7-11ea-9d7b-96398c507189.png)
no, it will not.

Reply:
I am not getting that error...

Reply:
You are getting an error in lua_baseentity.cpp, that is not part of this PR

Reply:
appveyor disagrees. need to fix it or it won't pass CI

Reply:
huh... okay

Reply:
I changed an `&&` to `and` to show that it throws an error. . .

Reply:
I don't think you're wrong. That will compile on newer VC++ compilers, but the solution for DSP hasn't been updated so it will fail on appveyor. That being said, we use `&&` in C++ and you should still stick to that for consistency sake.

Reply:
I dunno what to tell you. "and" and "&&" should be equivalent in modern C++.

Unless there is some windows / linux different here?

Reply:
> I would really like it if you would get this compiling, boot it up, and make sure raising and lowering merits work. Also double check the caps.

double check caps? What do you mean?

Reply:
Sorry, the merit point caps for each category. They didn't change at all, right? It's still 10 for group 2 and a max of 5 per merit?

Reply:
> I dunno what to tell you. "and" and "&&" should be equivalent in modern C++.

Consistency. You'll pop my brain if I have to notice 2 diff styles in same cpp file. :cry:

Reply:
Well, we definitely prefer whats left of your brain intact :P

Reply:
> Sorry, the merit point caps for each category. They didn't change at all, right? It's still 10 for group 2 and a max of 5 per merit?

Let me check group hp/mp, and stats, that might have changed. The others have not though.